build.gradle

```diff
@@ -146,3 +146,25 @@ check.dependsOn integrationTest
 task wrapper(type: Wrapper) {
     gradleVersion = '2.3'
 }
+
+//---------------------------Deploy in local Tomcat--------------------------------------------------//
+
+def tomcat_home = System.getenv('CATALINA_HOME')
+def tomcat_webapps = tomcat_home+'/webapps'
+def tomcat_bin = tomcat_home+'/bin'
+
+task deployToTomcat(type: Copy) {
+    from war
+    into tomcat_webapps
```

Comment:
Don't upload the war with a .war extension. Tomcat will attempt to start it before it's done copying, fail, and you will have to restart tomcat.
So u have 2 options:
1) You can call stopTomcat task before coping war file and call startTomcat after the copping will be done.
2) If tomcat instance have already running u can upload war in some tmp file({warName}.war.new). Remove old war file. Activate new war by removing `.new` suffix. More info [here](http://cadrlife.blogspot.com/2013/07/simple-war-deploy-to-tomcat-with-gradle.html).